scm/github.go

```diff
@@ -85,6 +69,22 @@ func (s *GithubSCM) GetOrganization(ctx context.Context, opt *GetOrgOptions) (*q
 		}
 	}
 
+	org := &qf.Organization{
```

Comment:
We can return `org` on Line 105, avoiding to "allocate" another `qf.Organization` object.